### ISSUE TITLE
invisible itemframes

### ISSUE THREAD

**Issue by oddlama** [oddlama/vane#121]
have a way to make item-frames invisible. maybe by right-clicking it with shears like in [tweakin](https://www.spigotmc.org/resources/tweakin-1-17-1-18.93444/). Definitely allows for some cool builds!

**Comment by FordFriedel** [comment]
On top of this, my number 1 gripe with item frames is how much lag they cause when many are spawned in.  I would like to look into how to reduce their impact or create a new item that is 2 dimensional instead of 3

**Comment by oddlama** [comment]
That's true but I'm not sure whether a more performant solution is possible without client mods.

**Comment by synuhaan** [comment]
Something i found and that would be cool, is dyable item-frames. St the moment, you can find datapacks (which don't work on my server for some reason). There you could shift-rightclick with a dye, and the datapack would place a wool-carpet in an armor stand inside of the item-frame.
I really love it and it might be a great addition.

I'm not sure if it's better to do it with an armor stand or if custom textures for the item frame itself would be an better option (for the lagg, if possible)

Anyways, what do you think?

**Comment by noahboa27** [comment]
@oddlama I would like to start working on this one. However, I don't know if dyable frames are still wanted and if so I think it should be a separate issue.

**Comment by oddlama** [comment]
@noahboa27 Of course! I'd be delighted if you want to work in this.

Invisible itemframes are definitely still wanted and shouldn't be too complicated to implement. Dying is of course still an option but I think it would require serious "hacks" to get it to work. We can definitely offload that into its own issue.
